### PR TITLE
Shouldn't say "Average" when it's less than a week

### PR DESCRIPTION
**Pull request by beweinreich**
The hourly peak chart shouldn't say "Average" if its just for a week.

### CODE REVIEW

File: src/helpers/multi-week-selection/index.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import moment from 'moment';
+
+export default function isMultiWeekSelection(startDate, endDate) {
+	let multiWeekSelection = false;
+	const startDateMoment = moment(startDate);
```
Comment (1egoman) [inline src/helpers/multi-week-selection/index.ts]: Keep in mind `moment()` uses the local time zone of the user for this calculation. I don't know if that's what you intended (I suspect not).

File: src/helpers/multi-week-selection/index.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+	const endDateMoment = moment(endDate);
+	if (endDateMoment.diff(startDateMoment, 'days') > 7) {
+		multiWeekSelection = true;
+	}
+	return multiWeekSelection;
```
Comment (1egoman) [inline src/helpers/multi-week-selection/index.ts]: You could probably simplify this to `return endDateMoment.diff(startDateMoment, 'days') > 7`

Comment (beweinreich) [reply src/helpers/multi-week-selection/index.ts]: good call.

File: src/helpers/multi-week-selection/index.ts (hunk 1 above)
Comment (beweinreich) [reply src/helpers/multi-week-selection/index.ts]: thats fine.

**Pull request merged**